modules/masterbar.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Module Name: WordPress.com Toolbar
+ * Module Description: An admin bar providing similar experience to the front-end navigation on WordPress.com.
+ * Sort Order: 38
+ * Recommendation Order: 16
+ * First Introduced: 4.7
+ * Requires Connection: Yes
+ * Auto Activate: No
+ * Module Tags: General
+ * Additional Search Queries: adminbar, masterbar
+ */
+
+include dirname( __FILE__ ) . '/masterbar/masterbar.php';
+
+new A8C_WPCOM_Masterbar;
```

Comment:
Since we're loading quite a few files from WordPress.com, maybe we should prefetch the domains we use for those resources, like we do for other modules? Here is an example with the Comments module:
https://github.com/Automattic/jetpack/blob/master/modules/comments.php#L33-L42